src/components/Typography/SwTypography.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <component
+    :is="getComponent"
+    :class="[variant]"
+    :style="{
+      'font-weight': fontWeight,
+      'font-family': `var(--${fontFamily})`,
+    }"
+  >
+    <slot></slot>
```

Comment:
why do we need slot here?

Reply:
Because you want the content of this tag to be styled

Reply:
oh now I see it, got ya